src/iface-ethernet-ip.c

```diff
@@ -206,6 +206,34 @@ static void attempt_connect(struct l_timeout *to, void *user_data)
 	l_timeout_modify(to, RECONNECT_TIMEOUT);
 }
 
+static int put_tag_name_created(struct knot_data_item *data_item)
+{
+	struct knot_data_item *data_item_aux;
+	int i;
+	int rc = -1;
+
+	for (i = 0; i < thing_ethernet_ip.number_sensor; i++) {
+		int return_aux;
+
+		data_item_aux = l_hashmap_lookup(thing_ethernet_ip.data_items,
+						 L_INT_TO_PTR(i));
+		if (!data_item_aux)
+			rc = -EINVAL;
+
+		return_aux = strcmp(
+			data_item_aux->tag_name,
+			data_item->tag_name);
+		if (!return_aux) {
+			data_item_aux->tag =
+				data_item->tag;
+			rc = 0;
```

Comment:
I don't understand the meaning of this number -1, I would recommend describing it.